tests/periph_uart/tests/01__periph_uart_base.robot

```diff
@@ -16,64 +16,73 @@ Variables           test_vars.py
 Force Tags          periph  uart
 
 *** Test Cases ***
-Echo
+Short Echo Should Succeed
+    [Documentation]     Write short string to UART and verify echo reply.
     PHILIP.Setup Uart
     UART Init and Flush Should Succeed
     API Call Should Succeed     Uart Write  ${SHORT_TEST_STRING}
     API Result Data Should Contain  ${SHORT_TEST_STRING}
-    Show PHILIP Statistics
+    Log                 PHILIP Stats
```

Comment:
It appears that this just logs the string `PHILIP Stats` and not any values

Reply:
mhm, that's weird